securedrop_client/conversation/transcript/transcript.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+from securedrop_client import db as database
+
+from .items import Item
+from .items import transcribe as transcribe_item
+
+
+def transcribe(record: database.Base) -> Optional[Item]:
```

Comment:
Note that `database.Base` is very broad, but the `transcript.Item` does the filtering quite naturally, so I think that's actually nice.